src/Concerns/HandlesGraphqlRequests.php

```diff
@@ -100,11 +101,22 @@ public function errorFormatter(GraphqlError $graphqlError)
             $throwable instanceof Exception ? $throwable : $graphqlError
         );
 
+        if ($throwable instanceof HttpException) {
```

Comment:
```suggestion
        if ($throwable instanceof HttpException && $throwable->getStatusCode() >= 400 && $throwable->getStatusCode() < 500) {
```

```suggestion
        if ($throwable instanceof HttpException && in_array($throwable->getStatusCode(), range(400, 499))) {
```

Something like that to ensure client errors only?

Reply:
Done.